solrsearch-plugins/src/main/java/co.cask.hydrator.plugin/common/SolrSearchSinkConfig.java

```diff
@@ -116,4 +92,95 @@ public String getIdField() {
     }
     return outputFieldMap;
   }
+
+  /**
+   * Returns the SolrClient for establishing the connection with Solr server, using the hostname and port provided by
+   * user.
+   *
+   * @return Solr client
+   */
+  public SolrClient getSolrConnection() {
+    String urlString;
+    SolrClient solrClient = null;
+    if (solrMode.equals(SINGLE_NODE_MODE)) {
```

Comment:
These values are coming from the ‘select widget’ on UI, which will always have the same case for its options.
Please let me know, if I still need to check values by ignoring case.